base/uk.ac.stfc.isis.ibex.client.tycho.parent/checkstyle.xml

```diff
@@ -1,99 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we no longer use the xml?

Reply:
we do sorry, i added it back in just in time